integrationtest/backend/elasticsearch/src/test/java/org/hibernate/search/integrationtest/backend/elasticsearch/testsupport/util/ElasticsearchTckBackendFeatures.java

```diff
@@ -299,4 +299,15 @@ public boolean supportsHighlighterFastVectorNoMatchSizeOnMultivaluedFields() {
 		// https://github.com/elastic/elasticsearch/issues/94550
 		return false;
 	}
+
+	@Override
+	public boolean supportsHighlighterPlainOrderByScoreMultivaluedField() {
+		// A plain highlighter implementation in ES had a bug
+		// https://github.com/elastic/elasticsearch/issues/87210
+		// that is now fixed with https://github.com/elastic/elasticsearch/pull/87414
+		return isActualVersion(
+				esVersion -> !esVersion.isBetween( "7.15", "8.4.0" ),
```

Comment:
I think the bug is fixed in 8.4, but this would assume it isn't?

```suggestion
				esVersion -> !esVersion.isBetween( "7.15", "8.3" ),
```

Reply:
good point! 😃